openapi/paths/API@bpm@caseInfo@{id}.yaml

```diff
@@ -3,6 +3,8 @@ get:
     - ProcessInstanceInfo
   summary: Finds the ProcessInstanceInfo by ID
   description: |
+    ![edition](https://img.shields.io/badge/edition-entreprise-blue)
```

Comment:
This is only true in the upcoming Bonita version. We should mention it here.

Reply:
You are right. I just did